docs/antora-ccutil/ccutil/guides-operator/master-docinfo.xml

```diff
@@ -0,0 +1,25 @@
+<productname>{project_name_full}</productname>
+<productnumber>{project_versionDoc}</productnumber>
+<subtitle>For Use with {project_name_full} {project_versionDoc}</subtitle>
+<title>{adminguide_name}</title>
```

Comment:
Wrong title. Should be Operator guide right.

Reply:
resolved.